libs/ledger-live-common/src/exchange/swap/utils/index.ts

```diff
@@ -61,14 +61,13 @@ export const isRegistrationRequired = (provider: string): boolean => {
 };
 
 export const getProviderName = (provider: string): string => {
-  switch (provider) {
-    case "cic":
-      return provider.toUpperCase();
-    case "oneinch":
-      return "1inch";
-    default:
-      return provider.charAt(0).toUpperCase() + provider.slice(1);
-  }
+  const providerMap: Record<string, string> = {
```

Comment:
[Small observation - non-blocking]: We could move this providerMap outside of the function. Then it won't be re-created every time the fn is run